src/Tooltip/examples/events.js

```diff
@@ -0,0 +1,30 @@
+class Example extends React.Component {
+
+  state = { isOpen: false }
+
+  onClick = () => { this.setState({ isOpen: !this.state.isOpen }); }
+
+  render() {
+
+    const popover = {
+      position: 'top',
+      anchor: 'center',
+      content: 'Tooltip content'
+    };
+
+    const { onClick, state: { isOpen } } = this;
+
+    return (
+      <FlexView vAlignContent='center'>
+        <Tooltip popover={{ ...popover, event: 'hover' }}>
+          <span>Hover me!</span>
+        </Tooltip>
+        <Divider size='large' orientation='horizontal' />
+        <Tooltip popover={{ ...popover, isOpen }}>
+          <Button flat size='tiny' onClick={onClick} label='Click me!' />
+        </Tooltip>
```

Comment:
what about adding also an example of an always open `Tooltip`? Do you think it would be confusing?